tests/recordings/e2e_personal_access_token_create_get_revoke.json

```diff
@@ -0,0 +1,137 @@
+            "Accept": "application\/vnd.contentful.management.v1+json",
+            "Accept-Encoding": "gzip",
+            "Content-Type": "application\/vnd.contentful.management.v1+json"
+        },
+        "body": "{\"name\":\"Test access token\",\"scopes\":[\"content_management_read\"]}"
+    },
+    "response": {
+        "status": {
+            "http_version": "1.1",
+            "code": "201",
+            "message": "Created"
+        },
+        "headers": {
+            "Accept-Ranges": "bytes",
+            "Access-Control-Allow-Headers": "Accept,Accept-Language,Authorization,Cache-Control,Content-Length,Content-Range,Content-Type,DNT,Destination,Expires,If-Match,If-Modified-Since,If-None-Match,Keep-Alive,Last-Modified,Origin,Pragma,Range,User-Agent,X-Http-Method-Override,X-Mx-ReqToken,X-Requested-With,X-Contentful-Version,X-Contentful-Content-Type,X-Contentful-Organization,X-Contentful-Skip-Transformation,X-Contentful-User-Agent,X-Contentful-Enable-Experimental-Feature",
+            "Access-Control-Allow-Methods": "DELETE,GET,HEAD,POST,PUT,OPTIONS",
+            "Access-Control-Allow-Origin": "*",
+            "Access-Control-Expose-Headers": "Etag",
+            "Access-Control-Max-Age": "1728000",
+            "Cache-Control": "max-age=0",
+            "Content-Type": "application\/vnd.contentful.management.v1+json",
+            "Date": "Wed, 23 Aug 2017 17:10:13 GMT",
+            "ETag": "W\/\"38d53c6bf77436cbd96c477ba03b6184\"",
+            "Location": "https:\/\/api.contentful.com\/users\/me\/access_tokens\/6CwBPSYBSi5bPdx45JSvaA",
+            "Server": "Contentful",
+            "Strict-Transport-Security": "max-age=15768000",
+            "X-Content-Type-Options": "nosniff",
+            "X-Contentful-Request-Id": "5bbb9a3be49c25b23362d44c4575ac20",
+            "X-Frame-Options": "ALLOWALL",
+            "X-XSS-Protection": "1; mode=block",
+            "Connection": "keep-alive",
+            "Set-Cookie": "visid_incap_673446=9jaKirfTRcGkhDFO79bXY\/O2nVkAAAAAQUIPAAAAAAAXteuglcNGyO\/3yBn904AC; expires=Thu, 23 Aug 2018 08:11:11 GMT; path=\/; Domain=.contentful.com, nlbi_673446=1MvlW51kDA3yPQg26lKYhQAAAAAxIV47iQQosNPotToPcH3I; path=\/; Domain=.contentful.com, incap_ses_476_673446=eulQRfpPliWsufYJfxebBvS2nVkAAAAAHggWfaNoS\/BgT9tGyze0BA==; path=\/; Domain=.contentful.com",
+            "X-Iinfo": "6-2103564-2103574 NNNN CT(121 119 0) RT(1503508211596 144) q(0 0 2 0) r(4 4) U5",
+            "X-CDN": "Incapsula",
+            "Content-Encoding": "gzip",
+            "Transfer-Encoding": "chunked"
+        },
+        "body": "{\n  \"name\":\"Test access token\",\n  \"revokedAt\":null,\n  \"sys\":{\n    \"type\":\"PersonalAccessToken\",\n    \"id\":\"6CwBPSYBSi5bPdx45JSvaA\",\n    \"createdAt\":\"2017-08-23T17:10:13Z\",\n    \"updatedAt\":\"2017-08-23T17:10:13Z\"\n  },\n  \"scopes\":[\"content_management_read\"],\n  \"token\":\"CFPAT-0614d386673959037b4de26d2b4f6e57bc9c8de9fb43370e8c24f195c24a904a\"\n}\n\n"
```

Comment:
I hope you revoked them ...

Reply:
Of course, it's revoked in the same test (and recording) 🙂 
https://github.com/contentful/contentful-management.php/pull/65/files/215f4f0035136c86cd6b7e934b718e123786e79a#diff-0800305c0f5116e37da15e426f94fbabR96